src/utils/criteria.ts

```diff
@@ -38,6 +38,7 @@ export function elmLibraryToCriteria(
     return {
       id: shortid.generate(),
       label: `${labelTitle}: ${statement.name}`,
+      display: statement.name,
```

Comment:
I was kind of hoping we could do this without adding another field with duplicate content. It looks like both here and in `builderModelToCriteria` below the `statement` field gets the same value as this new `display` field. Is there any reason we can't use that field here?

Reply:
Oh yea, I didn't realize it was the same... I initially made it a new label though because I envisioned having some way to edit it from the PathwayList but if we don't want to give the user that possibility I can just pull the display from the statement

Reply:
I mean, it's a nice feature idea but it would be pretty low in the priority list. And it is nice to have a separate field name that clearly indicates it's for display 

But if we stuck with `statement` would that reduce this to just the 2 lines of:
```
transitionRef.current.condition?.description || selectedCriteria.display
```

becoming
```
transitionRef.current.condition?.description || selectedCriteria.statement
```

Reply:
Yes